jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/ContextHandler.java

```diff
@@ -272,8 +272,8 @@ public void dump(Appendable out, String indent) throws IOException
     {
         dumpObjects(out, indent,
             new ClassLoaderDump(getClassLoader()),
-            Dumpable.named("context " + this, _context),
-            Dumpable.named("handler attributes " + this, _persistentAttributes));
+            new DumpableAttributes("handler attributes", _persistentAttributes),
+            new DumpableAttributes("attributes", _context));
```

Comment:
Maybe it's unrelated, but shouldn't we also dump `_baseResource`?

Reply:
That should be in the toString of the ContextHandler that is part of the dump.... but I will check....

Reply:
yes the base is dumped already